src/StarcounterClientFiles/wwwroot/sys/starcounter.html

```diff
@@ -1,2 +1,33 @@
 <link rel="import" href="palindrom-client/palindrom-client.html">
-<link rel="import" href="/sys/starcounter-include/starcounter-include.html">
\ No newline at end of file
+<link rel="import" href="/sys/starcounter-include/starcounter-include.html">
+<script>
+(function() {
+
+    function updateCurrentLinks(currentHref) {
+        const nodes = document.querySelectorAll("a[href]");
+        let lastCurrent, nextCurrent;
+        nodes.forEach((node) => {
+            if (node.hasAttribute("aria-current")) {
+                lastCurrent = node;
+            }
+
+            if (node.href === currentHref) {
+                nextCurrent = node;
+            }
+        });
```

Comment:
Wouldn't be faster to hand this loop and filter to the browsers istead of doing it in the JS level?

```js
lastCurrent = document.querySelector("a[aria-current]")
nextCurrent = document.querySelector(`[href="${currentHref}"]`);
```

Reply:
Your script will only find the first instance, mine will find all instances. I think it is important to find all.

Regarding `[href="${currentHref}"]`, I think this makes the strong comparison by the attribute string and will not match all possible variants of a href, such as:

```html
<a href="/myapp/mypage">Absolute link</a>
<a href="../myapp/mypage">Relative link</a>
<a href="//host:8080/myapp/mypage">Host link</a>
<a href="https:///host:8080/myapp/mypage">Protocol link</a>
```